lib/Core/TxTree.cpp

```diff
@@ -2286,6 +2290,8 @@ std::string TxTree::getInterpolationStat() {
   printTimeStat(stream);
   stream << "KLEE: done: TxTreeNode method execution times (ms):\n";
   TxTreeNode::printTimeStat(stream);
+  // printing node count
+  TxTreeNode::printNodeCntStat(stream);
```

Comment:
It may look nicer if the nodes are printed at the start instead of at the end of the interpolation statistics.